specification/personal-demographics.yaml

```diff
@@ -75,13 +75,15 @@ info:
 
     ## API status and roadmap
     ### Application-restricted access
-    This access mode is in [beta](https://digital.nhs.uk/developer/guides-and-documentation/reference-guide#api-status), meaning:
-    - we might make breaking changes, but only if we cannot avoid it, and we'll give advance notice
+    This access mode is a [live](https://digital.nhs.uk/developer/guides-and-documentation/reference-guide#api-status) service, meaning:
```

Comment:
Should we be calling this [stable] or [live]?
Note HL7 says [stable] - but we refer to GDS through our links, which calls it [live]

Reply:
We deliberately changed to stable, which we explain as being "live and available for production use":
https://digital.nhs.uk/developer/guides-and-documentation/reference-guide#api-status 
Maybe this was because the beta API also gave you access to live data, so calling it just "live" was confusing.
I think that was the reason.